banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/bank_reconciliation_tool_beta.py

```diff
@@ -215,16 +207,33 @@ def create_payment_entry_bts(
 		return payment_entry  # Return saved document
 
 	payment_entry.submit()
+
+	return reconcile_entry(
+		bank_transaction_name, paid_amount, "Payment Entry", payment_entry.name
+	)
+
+
+@frappe.whitelist()
+def reconcile_entry(transaction_name, amount, voucher_type, voucher_name):
```

Comment:
```suggestion
def reconcile_voucher(transaction_name, amount, voucher_type, voucher_name):
```

Let's make the function name match the parameter names?